### DISCUSSION TITLE
Problem with rebuilding .html

### DISCUSSION THREAD

**Discussion by St4rBoyeK** [tailwindlabs/tailwindcss#10376] [Help]
My path is: `"C:\Users\mrkro\Desktop\terstyy\[xddd]\nui"`
cmd: `npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch`
![image](https://user-images.githubusercontent.com/40685883/213797448-3d458d52-3585-4847-ad8e-7d51d3633903.png)
![image](https://user-images.githubusercontent.com/40685883/213797706-c57294e3-7b01-4b1c-bb32-d8c1dabe38b5.png)
when I save index.html css isn't rebuilding, but when I save input.css it's rebuilding. I've noticed that this problem occurs when subfolders contain "[]" characters. What can I do besides renaming the folders?

**Comment by St4rBoyeK** [comment]
bump

**Comment by wongjn** [comment]
You seem to have a mistake in the `content` file content glob: it seems you are after `./src/**/*.html` (or `src/**/*.html`) but you have `.src/**/*.html` (notice the missing slash after the beginning dot).

**Reply by wongjn** [reply]
> mb. I have ` content: ["./src/**/*.{html,js}"],` and it still not working[^1]

You could try updating to the latest Tailwind if you have not already. I believe there have been some fixes like #9650 to fix special characters in the working directory path. Otherwise, consider trying a path with no special characters temporarily to diagnose if it is indeed the fact that the special characters is causing the undesired behavior.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/10376#discussioncomment-5114094

**Reply by St4rBoyeK** [reply]
Nope, it didn't help. I will start using a path with no special characters. Thanks anyway

**Comment by St4rBoyeK** [comment]
mb. I have ` content: ["./src/**/*.{html,js}"],` and it still not working

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/10376#discussioncomment-5114124 to keep the reply chain together.
